packages/components/psammead-navigation/src/DropdownNavigation/index.test.jsx

```diff
@@ -73,6 +75,7 @@ describe('Canonical', () => {
         onOpen={() => {}}
         isOpen
         onClose={() => {}}
+        script={latin}
```

Comment:
can we have at least one test where script isn't `latin`?

Reply:
I've added Arabic rtl snapshot tests for each bit of the dropdown.